srcpkgs/consul/files/consul/run

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+exec 2>&1
+exec consul agent -config-dir=/etc/consul.d
```

Comment:
I'd feel better if this config dir was configurable. Maybe just have all the flags configurable with a sane default.

Reply:
Fair points.